src/sub-blocks/PriceDetailed/PriceDetails/Details/List.tsx

```diff
@@ -20,6 +20,8 @@ const List = (props: ListProps) => {
         <Fragment>
             {items.map((item, id) => (
                 <div key={id} className={b('list-item')}>
+                    {/* TODO fix in https://github.com/gravity-ui/page-constructor/issues/964*/}
+                    {/* eslint-disable-next-line jsx-a11y/alt-text */}
                     <img className={b('img')} />
```

Comment:
it seems this `<img>` tag corresponds to the mark icon (see screenshot). It plays only decorative role so we can set `alt=""`
<img width="499" alt="Screenshot 2024-07-31 at 12 11 50" src="https://github.com/user-attachments/assets/d16b5772-5963-4a99-89d6-ecd88e25e0b2">

Reply:
done